include/openssl/tls1.h

```diff
@@ -30,14 +30,6 @@ extern "C" {
 # define TLS1_3_VERSION                  0x0304
 # define TLS_MAX_VERSION                 TLS1_3_VERSION
 
-/* TODO(TLS1.3) REMOVE ME: Version indicators for draft version */
-# define TLS1_3_VERSION_DRAFT_26         0x7f1a
-# define TLS1_3_VERSION_DRAFT_27         0x7f1b
-# define TLS1_3_VERSION_DRAFT            0x7f1c
-# define TLS1_3_VERSION_DRAFT_TXT_26     "TLS 1.3 (draft 26)"
-# define TLS1_3_VERSION_DRAFT_TXT_27     "TLS 1.3 (draft 27)"
-# define TLS1_3_VERSION_DRAFT_TXT        "TLS 1.3 (draft 28)"
```

Comment:
What's the harm in keeping these around?  They'll still be around in the wild for a little while, at least.

Reply:
I do not think they will be around very long, and this is an LTS release.  I think the right decision is to remove them.

Reply:
I am strongly opposed to keeping these around. Once they're in a stable release they will be with us for a very long time. As Rich says this is an LTS release. Lets get rid of the excess baggage now while we can. I don't think support for this in the wild will be around for very long - and we should do everything we can to encourage people to switch to the final version asap.

Reply:
I agree. Remove the baggage. We are only supporting the RFC version not earlier drafts.